File: trunk/Design/SoftwareIdeasModeler/Data/Templates/Code/VbNet/Default/Enum.vb
<%==SIM:Set:private=Private==%>
<%==SIM:Set:protected=Protected==%>
<%==SIM:Set:package=Friend==%>
<%==SIM:Set:public=Public==%>
<%==SIM:ForEach:Imports==%>
Imports <%==SIM:Import.Name==%>
<%==SIM:EndFor==%>

Namespace <%==SIM:Element.Namespace==%>
    <%==SIM:ForEach:Element.DocumentationLines==%>'<%==SIM:DocumentationLine==%>
    <%==SIM:EndFor==%><%==SIM:Element.Visibility==%> Enum <%==SIM:Element.Name==%>

<%==SIM:ForEach:Element.Attributes==%>
	    <%==SIM:Attribute.Name==%> <%==SIM:If:Attribute.HasDefaultValue==%> = <%==SIM:Attribute.DefaultValue==%><%==SIM:EndIf==%>
<%==SIM:EndFor==%>

    End Enum
End Namespace

File: Design/SoftwareIdeasModeler/Data/Templates/Code/Java/Default/Enum.java
package <%==SIM:Element.Namespace==%>;

<%==SIM:Set:private=private==%>
<%==SIM:Set:protected=protected==%>
<%==SIM:Set:package= ==%>
<%==SIM:Set:public=public==%>
<%==SIM:ForEach:Imports==%>
import <%==SIM:Import.Name==%>;
<%==SIM:EndFor==%>

<%==SIM:ForEach:Element.DocumentationLines==%>//<%==SIM:DocumentationLine==%>
<%==SIM:EndFor==%><%==SIM:Element.Visibility==%> enum <%==SIM:Element.Name==%>
{
<%==SIM:ForEach:Element.Attributes==%>
    <%==SIM:ForEach:Attribute.DocumentationLines==%>//<%==SIM:DocumentationLine==%>
    <%==SIM:EndFor==%><%==SIM:Attribute.Name==%>,
<%==SIM:EndFor==%>
}
